apps/backend/src/resolvers/types/Fap.ts

```diff
@@ -65,26 +65,30 @@ export class FapResolvers {
     );
   }
 
-  @FieldResolver(() => BasicUserDetails, { nullable: true })
+  @FieldResolver(() => [BasicUserDetails])
   async fapSecretary(@Root() fap: Fap, @Ctx() context: ResolverContext) {
-    if (!fap.fapSecretaryUserId) {
-      return null;
+    if (!fap.fapSecretaryUserIds) {
+      return [];
     }
 
-    return context.queries.user.getBasic(context.user, fap.fapSecretaryUserId);
+    return fap.fapSecretaryUserIds.map((fapSecretaryUserId) =>
+      context.queries.user.getBasic(context.user, fapSecretaryUserId)
+    );
   }
 
-  @FieldResolver(() => Int, { nullable: true })
+  @FieldResolver(() => [Int])
   async fapSecretaryProposalCount(
     @Root() fap: Fap,
     @Ctx() context: ResolverContext
   ) {
-    if (!fap.fapSecretaryUserId) {
-      return null;
+    if (!fap.fapSecretaryUserIds) {
```

Comment:
With it just returning a list of integers, is this method useful? Would it be better to return a list of objects like `{userId,count}` so the numbers can have context attached?